GOVERNANCE.md

```diff
@@ -0,0 +1,127 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release, Long Term Support (LTS) and
+Canary in the Gold Mine (CitGM) teams. It manages the release
+and long term support schedule and policies for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
```

Comment:
(Also: I read this as delegating it entirely to the Release WG and I am 👍 on that. If people want to have input on release schedules, then the Release WG is the place to be.)